lib/livebook/live_markdown/export.ex

```diff
@@ -336,7 +336,13 @@ defmodule Livebook.LiveMarkdown.Export do
   defp render_notebook_footer(notebook, notebook_source, true) do
     metadata = notebook_stamp_metadata(notebook)
 
-    case Livebook.Hubs.fetch_hub(notebook.hub_id) do
+    result =
+      case Livebook.Hubs.fetch_offline_hub(notebook.hub_id) do
+        {:ok, _} = result -> result
+        :error -> Livebook.Hubs.fetch_hub(notebook.hub_id)
+      end
```

Comment:
```suggestion
      with :error <- Livebook.Hubs.fetch_offline_hub(notebook.hub_id) do
        Livebook.Hubs.fetch_hub(notebook.hub_id)
      end
```

Reply:
Would an offline hub be used to stamp? We only use it to validate stamps, no?

Reply:
We only use to validate the stamp. I'll remove since we don't plan to keep the stamp if the session saves the notebook which has the offline hub